src/saveload/compat/company_sl_compat.h

```diff
@@ -97,6 +97,8 @@ const SaveLoadCompat _company_sl_compat[] = {
 	SLC_VAR("face"),
 	SLC_VAR("money"),
 	SLC_VAR("current_loan"),
+	SLC_VAR("auto_loan"),
+	SLC_VAR("auto_loan_threshold"),
```

Comment:
You shouldn't add this here, this section is compatibility with saves with unnamed fields, which will never include these new fields.